migrator.sh

```diff
@@ -347,6 +347,47 @@ decode_auth() {
   fi
 }
 
+##
+# Get all the tags for a name at the target repository
+#
+# Usage: query_tags_at_target <image_name>
+# Return: json-list of all tags in the form ["latest", "dev", "123"]. If the request fails, the list will be empty
+##
+query_tags_at_target() {
+    IMAGE="${1}"
+
+    TAGS_URL="https://${V2_REGISTRY}/v2/${IMAGE}/tags/list"
```

Comment:
I'd suggest using `${V2_PROTO}` instead of `https` since it is possible for an insecure v2 to be used.